keda/values.yaml

```diff
@@ -24,6 +24,26 @@ image:
 # -- Kubernetes cluster domain
 clusterDomain: cluster.local
 
+verticalPodAutoscaler:
+  keda:
+    # -- Enables the creation of a vpa for the keda-operator
+    enabled: true
```

Comment:
I think this should be false by default upstream

Reply:
Done